.github/workflows/publish.yml

```diff
@@ -54,22 +54,12 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref || '' }}
-
-      - name: Gather image info
-        id: info
-        run: |
-          echo "repo-owner=${GITHUB_REPOSITORY_OWNER,,}" >> $GITHUB_OUTPUT
```

Comment:
This step was making sure `github.repository_owner` was consistently expressed in lower case.

Reply:
Hey @WadeBarnes, thank you for the review, I have added it back with a comment explaining what it does.